Makefile

```diff
@@ -77,7 +80,7 @@ a11y: test-build pally-conf
 
 # Note: `run` executes `node demo/app`, which fires up express, then deploys
 # a test static site to s3, then exits, freeing the process to execute `nightwatch a11y`.
-test: developer-note verify pally-conf test-server test-browser test-build test-webpack run nightwatch a11y
+test: developer-note verify pally-conf test-server test-browser test-build test-webpack run nightwatch a11y build-webpack
```

Comment:
@leggsimon `test` task should absolutely not generate anything for deployment... it should only test
I think all developers are clear about that
Anything generated for deployment should be in the `build` step, never `test`.
If you see any project that generates/builds in the `test` step then that project should really move it into `build`.